### PR TITLE
[Ops] Added CI/CD workflows with GitHub actions

### PR DESCRIPTION
**Pull request by naman03malhotra**

### CODE REVIEW

**Review by s-yadav — APPROVED** [review]

File: .github/workflows/CD.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+jobs:
+  publish:
+    name: Package publish using node ${{ matrix.node }}
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        node: [14]
```
Comment (ParthS007) [inline .github/workflows/CD.yml:16]:
```suggestion
        node: ['14']
```

File: .github/workflows/CD.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+name: CD
+
+on:
+  push:
+    branches:
+      - master
```
Comment (ParthS007) [inline .github/workflows/CD.yml:6]: what about running `CD` only when pushed with a tag?

**Review by ParthS007 — COMMENTED** [review]
Thanks for the PR. Left some minor suggestions.

File: .github/workflows/CI.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  test:
+    name: Test on node ${{ matrix.node }} and ${{ matrix.os }}
+
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        node: [14]
+        os: [ubuntu-latest]
```
[the remark this answers is not in the document]
Comment (naman03malhotra) [reply .github/workflows/CI.yml:22]: For some reason, this was not working, can you please create a PR for your suggestions.

**Pull request merged**